src/errors/checkXcmTxInputs.ts

```diff
@@ -1,74 +1,221 @@
-import { SYSTEM_PARACHAINS_IDS } from '../consts';
+import { RELAY_CHAIN_IDS, SYSTEM_PARACHAINS_IDS } from '../consts';
 import { findRelayChain } from '../registry/findRelayChain';
 import type { ChainInfo, ChainInfoRegistry } from '../registry/types';
-import type { Direction } from '../types';
+import { Direction } from '../types';
 import { BaseError } from './BaseError';
 
 /**
- * This will check the given assetIds and ensure they are either valid integers as strings
- * or known token symbols
+ * This will check that a given assetId is neither an empty string
+ * or known blank space
  *
- * @param assetIds
+ * @param assetId
+ */
+const checkIfAssetIdIsEmptyOrBlankSpace = (assetId: string) => {
+	// check if empty or space
+	// if assetId is an empty space or space error
+	if (assetId === '' || assetId === ' ') {
```

Comment:
```suggestion
	if (assetId === '' || assetId.trim() === '') {
```

This will ensure that we cover all cases where blank spaces can be any size.